tools/sphinx_ext/run_examples.py

```diff
@@ -0,0 +1,401 @@
+from __future__ import annotations
```

Comment:
Please, add a note that we took some parts of it from `Litestar`. Let's credit where credit is due.